mods/d2k/maps/atreides-06a/atreides06a.lua

```diff
@@ -0,0 +1,237 @@
+	end
+
+	if Ordos.HasNoRequiredUnits() and OrdosSmall.HasNoRequiredUnits() and not OrdosKilled then
+		Media.DisplayMessage(UserInterface.GetFluentMessage("ordos-annihilated"), Mentat)
+		OrdosKilled = true
+		Atreides.MarkCompletedObjective(KillOrdos)
+	end
+
+	if Smugglers.HasNoRequiredUnits() and not SmugglersKilled then
+		Media.DisplayMessage(UserInterface.GetFluentMessage("smugglers-annihilated"), Mentat)
+		SmugglersKilled = true
+		Atreides.MarkCompletedObjective(KillSmugglers)
+	end
+
+	if DateTime.GameTime % DateTime.Seconds(10) == 0 and LastHarvesterEaten[Ordos] then
+		local units = Ordos.GetActorsByType("harvester")
+
+		if #units > 0 then
+			LastHarvesterEaten[Ordos] = false
+			ProtectHarvester(units[1], Ordos, AttackGroupSize[Difficulty])
+		end
+	end
+
+
+end
+
+WorldLoaded = function()
+
+	Ordos = Player.GetPlayer("Ordos")
+	Smugglers = Player.GetPlayer("Smugglers")
+	OrdosSmall = Player.GetPlayer("OrdosSmall")
+	Atreides = Player.GetPlayer("Atreides")
+	Ordos.Cash = 12000
+	OrdosSmall.Cash = 5000
+	Smugglers.Cash = 3000
+	InitObjectives(Atreides)
+	KillOrdos = AddPrimaryObjective(Atreides, UserInterface.GetFluentMessage("destroy-ordos"))
+	KillOrdosStarport = AddPrimaryObjective(Atreides, UserInterface.GetFluentMessage("destroy-ordos-starport"))
+	KillSmugglers = AddSecondaryObjective(Atreides, UserInterface.GetFluentMessage("destroy-smugglers"))
+	KillAtreides1 = AddPrimaryObjective(Ordos, "")
+	KillAtreides2 = AddPrimaryObjective(OrdosSmall, "")
+
+	Camera.Position = AMCV.CenterPosition
+	AttackLocation = AMCV.Location
+
+	Trigger.OnAllKilledOrCaptured(OrdosMainBase, function()
+		Utils.Do(Ordos.GetGroundAttackers(), IdleHunt)
+	end)
+
+	Trigger.OnAllKilledOrCaptured(SmugglersBase, function()
+		Utils.Do(Smugglers.GetGroundAttackers(), IdleHunt)
+	end)
+
+	Trigger.OnAllKilledOrCaptured(OrdosSmallBase, function()
+		Atreides.MarkCompletedObjective(KillOrdosStarport)
+		Media.DisplayMessage(UserInterface.GetFluentMessage("ordos-starport-destroyed"), Mentat)
+		Utils.Do(OrdosSmall.GetGroundAttackers(), IdleHunt)
+	end)
+
+	local path = function() return Utils.Random(OrdosReinforcementsPaths) end
+	local waveCondition = function() return OrdosKilled end
+	local huntFunction = function(unit)
+		unit.AttackMove(AttackLocation)
+		IdleHunt(unit)
+	end
+	SendCarryallReinforcements(Ordos, 0, OrdosAttackWaves[Difficulty], OrdosReinforcementsDelay[Difficulty], path, OrdosReinforcements[Difficulty], waveCondition, huntFunction)
+
+	SendStarportReinforcements()
+	InitStarportReinforcements(DateTime.Minutes(3), InitialStarportSquad[Difficulty])
+	Actor.Create("upgrade.barracks", true, { Owner = Ordos })
+	Actor.Create("upgrade.light", true, { Owner = Ordos })
+	Actor.Create("upgrade.heavy", true, { Owner = Ordos })
+	Actor.Create("upgrade.barracks", true, { Owner = Smugglers })
+	Trigger.AfterDelay(0, ActivateAI)
+
+	SendAtreidesReinforcements(DateTime.Minutes(6), AtreidesReinforcements[Difficulty])
```

Comment:
```suggestion
	SendAtreidesReinforcements(DateTime.Minutes(6))
```
Unlike most of the others, this does not take a squad parameter.